cmd/vault-subpath-proxy/kv_update_transport.go

```diff
@@ -22,12 +28,16 @@ const secretKeyValidationRegexString = `^[a-zA-Z0-9\.\-_]+$`
 
 var secretKeyValidationRegex = regexp.MustCompile(secretKeyValidationRegexString)
 
-type kvKeyValidator struct {
+type kvUpdateTransport struct {
```

Comment:
Not _that_ great of a name but the best I could come up with, very open for better suggestions